src/react/components/shell/mainContentRouter.tsx

```diff
@@ -12,6 +12,7 @@ import ProfileSettingsPage from "../pages/profileSettingsPage";
 export default function MainContentRouter() {
     return (
         <div className="app-content">
+            {window.location.pathname.includes('index.html') && <Redirect to="/" />}
```

Comment:
Instead of hard coding in `index.html` can we introduce a catch all route that redirects to the homepage?

Reply:
I'd also like to introduce some unit tests to verify this doesn't sneak back up on us at some point.

Reply:
Looks like you also have a minor ts linting issue.

Reply:
fixed, fixed, and fixed.